pages/dkp/kommander/2.2/release-notes/index.md

```diff
@@ -24,17 +24,14 @@ This version supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster
 
 ### New features and capabilities
 
-#### Feature 1, 2, etc.
+#### Upgrade catalog application via CLI and UI
 
+You can use either the CLI or the UI to [upgrade your catalog applications](..\projects\applications\catalog-applications\index.md).
```

Comment:
Let's add a note somewhere in the release notes that catalog apps **must** be upgraded to the latest version BEFORE upgrading the kubernetes version (or konvoy version for managed konvoy clusters) on attached clusters, due to the previous catalog app versions incompatibility with k8s 1.22

Reply:
Thanks Grace, I'll add this to that new section of the RN now.